README.md

```diff
@@ -71,8 +67,6 @@ Alternatively you can provide the location on command invocation using the `--sc
 
 ```shell
 yarn prisma --schema node_modules/@yes-theory-fam/database/prisma/schema.prisma
-# or using npm
-npm run prisma -- --schema node_modules/@yes-theory-fam/database/prisma/schema.prisma
```

Comment:
On second thought: This could be left in, as this is documentation about how _this package_ can be used from _another project_, in which case both yarn and npm could be used.